Makefile

```diff
@@ -8,12 +8,36 @@ VERSION := $(shell git describe --tags --always --dirty="-dev")
 LDFLAGS := -ldflags='-X "main.Version=$(VERSION)"'
 
 test:
-	GO111MODULE=on go test -mod=vendor -v ./...
-
-all: dist/aws-okta-$(VERSION)-darwin-amd64 dist/aws-okta-$(VERSION)-linux-amd64
+	GO111MODULE=on go test -mod=vendor -covermode=count -coverprofile=coverage.out -v ./...
```

Comment:
General comment: try to keep the PR focussed.  While this change is ok, it should have been in a separate PR. I know your intentions are good, but it makes it harder to accept the PR when there's orthogonal changes.

Reply:
for these types of changes I can pull then out to another PR. I needed the changes while writing tests so I included it, it's not needed though.